src/worker.ts

```diff
@@ -31,6 +31,12 @@ function getConfig(url: string): Promise<Config> {
 
 export function init({ k: apiToken }: QueryParameters): Promise<Beacon[]> {
   const configUrl = `${CONFIG_URL}${apiToken}`;
-  return getConfig(configUrl).then(runTasks);
-  // TODO: deal with errors
+  return getConfig(configUrl)
+    .then(runTasks)
+    .catch(
+      (): Promise<Beacon[]> => {
+        // TODO: deal with errors
+        return Promise.resolve([]);
```

Comment:
You should be able to simplify to `return []` if you like

Reply:
Hmmm `tsc` didn't seem to agree (whilst I do):

```Type 'never[]' is missing the following properties from type 'Promise<Beacon[]>': then, catch, [Symbol.toStringTag], finally```

Reply:
Even if you drop the `():Promise<Beacon[]>=>{` declaration?